### PR TITLE
Update share and share target articles for M89 support.

### LINKED ISSUE #4496: content: Update web share articles
* Use this template to request updates to existing content.
* Delete any fields that aren't relevant to your issue. 
* Update the title of this issue with a brief summary of the work to be done.

### What page(s) need to be updated?

* [Integrate with the OS sharing UI with the Web Share API](https://web.dev/web-share/)
* [Receiving shared data with the Web Share Target API](https://web.dev/web-share-target/)

### Why is this update needed?

Web Share and Web Share Target are now available on desktop. Previously they were only available on Android. 

### What's the deadline?

January 27, 2021

#### Is it a hard deadline?

Yes. It's being announced in the beta release post which will be published the following day.

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #4496

Changes proposed in this pull request:

Add mention of new support for desktop platforms in Chrome 89.

### CODE REVIEW

File: src/site/content/en/blog/web-share/index.md [hunk 1]
```diff
@@ -47,8 +47,9 @@ Web share has the following capabilities and limitations:
 * It must be invoked in response to a user action such as a click. Invoking it
   through the `onload` handler is impossible.
 * It can share, URLs, text, or files.
-* As of mid 2020, it is only available on Safari and on Android in Chromium
-  forks. See
+* As of January 2021, it is available on Android in Chromium forks, Chrome OS,
+  and Chrome for Windows. Chrome for Safari on MacOS is still in development.
```
Comment (beaufortfrancois) [inline src/site/content/en/blog/web-share/index.md:51]: I assume you meant `Chrome on macOS` and `Chrome on Windows`. Not safari ;)

**Review by beaufortfrancois — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/web-share/index.md (hunk 1 above)
Comment (jpmedley) [reply src/site/content/en/blog/web-share/index.md:51]: It's apparently still in development on MacOS. See the [status entry comments](https://www.chromestatus.com/feature/5668769141620736).

Comment (beaufortfrancois) [reply src/site/content/en/blog/web-share/index.md:51]: I meant "Chrome on Windows", not "Chrome for Windows".
And not "Chrome for Safari on macOS", but just "Chrome on macOS" as Safari is another browser from another company ;)

Comment (jpmedley) [reply src/site/content/en/blog/web-share/index.md:51]: I'm having trouble getting this in my head. Can you do this as a suggestion, please.

Comment (beaufortfrancois) [reply src/site/content/en/blog/web-share/index.md:51]: Here's what I would write.
```
* As of January 2021, it is available on Safari, on Android in Chromium forks,
  on Chrome OS, and Chrome on Windows. Chrome on MacOS is still in development.
  ```

**Comment by kaycebasques** [comment]
Looks like next step is for @jpmedley to resolve the last outstanding item that @beaufortfrancois mentioned

**Comment by kaycebasques** [comment]
Hello! This is an automated review by our custom [reviewbot](https://github.com/kaycebasques/reviewbot). It updates automatically when code or GitHub comments in this pull request are created or updated.

## Staging URLs

For your convenience, here are direct links (on our staging site) to the content you created or updated:

* https://deploy-preview-4497--web-dev-staging.netlify.app/web-share-target/
* https://deploy-preview-4497--web-dev-staging.netlify.app/web-share/
## Requested changes

If there are any common problems with the content files you created or modified, they will be listed here.

### `src/site/content/en/blog/web-share-target/index.md`

* This file passed all of our automated Markdown audits.

### `src/site/content/en/blog/web-share/index.md`

* This file passed all of our automated Markdown audits.

File: src/site/content/en/blog/web-share/index.md (hunk 1 above)
Comment (jpmedley) [reply src/site/content/en/blog/web-share/index.md]: Done. Please take another look.

**Review by beaufortfrancois — APPROVED** [review]
Lgtm

**Review by kaycebasques — APPROVED** [review]
@jpmedley it seems like you should bump the `updated` fields for these 2 docs but other than that this LGTM

**Pull request merged**